client/my-sites/site-settings/amp/jetpack.jsx

```diff
@@ -29,6 +29,9 @@ const AmpJetpack = ( {
 } ) => {
 	let linkUrl, linkText;
 	if ( ampPluginInstalled && ampPluginInstalled.active ) {
+		if ( ! customizerAmpPanelUrl ) {
+			return null;
```

Comment:
The problem with bailing out here is that the  `<QueryJetpackPlugins />` component below will not fire its request.

Reply:
Maybe we should add to the existing check further down:
`{ ! requestingPlugins && <CompactCard href={ linkUrl }>{ linkText }</CompactCard> }`